frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/PaymentsInputPanel.tsx

```diff
@@ -475,7 +475,13 @@ export const PaymentsInputPanel = (): JSX.Element | null => {
   }, [paymentsField, resetData, setData, setToEditingPayment, setToInactive])
 
   const paymentDisabledMessage = !isEncryptMode ? (
-    <Text>Payments are not available in email mode forms.</Text>
+    <Text>
```

Comment:
nit: Might be easier to just say that payments is only available in storage mode so we don't always have to change this message when we add a new mode